pom.xml

```diff
@@ -28,6 +28,7 @@
         <module>csv-validator-cmd</module>
         <module>csv-validator-ui</module>
         <module>csv-validator-java-api</module>
+        <module>csv-validator-application</module>
```

Comment:
Would this be better named - `<module>csv-validator-distribution</module>` perhaps?

Reply:
done